params/config.go

```diff
@@ -490,6 +501,7 @@ func (c *ChainConfig) CheckConfigForkOrder() error {
 		{name: "cancunBlock", block: c.CancunCompatibleBlock},
 		{name: "randaoBlock", block: c.RandaoCompatibleBlock, optional: true},
 		{name: "kaiaBlock", block: c.KaiaCompatibleBlock},
+		{name: "pragueBlock", block: c.PragueCompatibleBlock, optional: true},
```

Comment:
Is this fork optional? Does it include CL as well?

Reply:
Opps, fixed at https://github.com/kaiachain/kaia/pull/105/commits/e4d328539a866d489e07238c26b9ae81ea767549 Thank you